# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../openlibraries/0001-libav-replace-deprecated_PIX_FMT_names.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Fix to replace deprecated PIX_FMT names by the newer variants

Reference:

http://git.libav.org/?p=libav.git;a=commitdiff;h=71e445fca3ccc1285068386bf9858d180b0fecfc

--- ./src/openmedialib/plugins/avformat/avformat_plugin.cpp.orig	2011-04-17 11:59:29.112077819 +0200
+++ ./src/openmedialib/plugins/avformat/avformat_plugin.cpp	2011-04-17 12:35:52.872071393 +0200
@@ -65,11 +65,11 @@
 		return L"yuv420p";
 	else if ( fmt == PIX_FMT_YUVJ420P )
 		return L"yuv420p";
-	else if ( fmt == PIX_FMT_UYVY411 )
+	else if ( fmt == PIX_FMT_UYYVYY411 )
 		return L"yuv411";
 	else if ( fmt == PIX_FMT_YUV411P )
 		return L"yuv411p";
-	else if ( fmt == PIX_FMT_YUV422 )
+	else if ( fmt == PIX_FMT_YUYV422 )
 		return L"yuv422";
 	else if ( fmt == PIX_FMT_YUV422P )
 		return L"yuv422p";
@@ -79,7 +79,7 @@
 		return L"r8g8b8";
 	else if ( fmt == PIX_FMT_BGR24 )
 		return L"b8g8r8";
-	else if ( fmt == PIX_FMT_RGBA32 )
+	else if ( fmt == PIX_FMT_RGB32 )
 		return L"r8g8b8a8";
 	return L"";
 }
